weblab/experiments/models.py

```diff
@@ -63,10 +63,26 @@ def get_name(self, model_version=False, proto_version=False):
     @property
     def visibility(self):
         return get_joint_visibility(
-            self.model.get_version_visibility(self.model_version, default=Visibility.PRIVATE),
-            self.protocol.get_version_visibility(self.protocol_version, default=Visibility.PRIVATE),
+            self.model.get_version_visibility(
+                self.model_version,
+                default=self.model.DEFAULT_VISIBILITY),
+            self.protocol.get_version_visibility(
+                self.protocol_version,
+                default=self.protocol.DEFAULT_VISIBILITY),
         )
 
+    @property
+    def viewers(self):
+        """
+        Should never be called if both are public
```

Comment:
I think this comment could be phrased better, it's just that it makes no sense to do so (since everybody is a viewer) and it won't be called by the higher level logic.